app/src/main/java/org/wikipedia/feed/news/NewsCard.java

```diff
@@ -1,25 +1,25 @@
 package org.wikipedia.feed.news;
 
 import androidx.annotation.NonNull;
-import androidx.annotation.VisibleForTesting;
 
 import org.wikipedia.R;
 import org.wikipedia.dataclient.WikiSite;
 import org.wikipedia.feed.model.CardType;
-import org.wikipedia.feed.model.ListCard;
 import org.wikipedia.feed.model.UtcDate;
+import org.wikipedia.feed.model.WikiSiteCard;
 import org.wikipedia.util.L10nUtil;
 
-import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
-public class NewsListCard extends ListCard<NewsItemCard> {
+public class NewsCard extends WikiSiteCard {
     @NonNull private UtcDate date;
+    @NonNull private List<NewsItem> news;
     @NonNull private WikiSite wiki;
 
-    public NewsListCard(@NonNull List<NewsItem> news, int age, @NonNull WikiSite wiki) {
-        super(toItemCards(news, wiki), wiki);
+    public NewsCard(@NonNull List<NewsItem> news, int age, @NonNull WikiSite wiki) {
+        super(wiki);
+        this.news = news;
         this.date = new UtcDate(age);
         this.wiki = wiki;
```

Comment:
This will be not necessary since you already have already extended the `WikiSiteCard`. Please see #1632